capybara-screenshot.gemspec

```diff
@@ -19,8 +19,6 @@ Gem::Specification.new do |s|
     "source_code_uri"   => "https://github.com/mattheworiordan/capybara-screenshot/tree/v#{s.version}",
   }
 
-  s.rubyforge_project = "capybara-screenshot"
-
```

Comment:
That's deprecated:
```
NOTE: Gem::Specification#rubyforge_project= is deprecated with no replacement. It will be removed on or after 2019-12-01.
Gem::Specification#rubyforge_project= called from /home/adrien/work/capybara-screenshot/capybara-screenshot.gemspec:22.
```